gulp/tasks/test.js

```diff
@@ -95,6 +94,18 @@ function _getProtractorParams() {
   return params;
 }
 
+function _getWCAGParams() {
+  var commandLineParams = minimist( process.argv.slice( 2 ) );
+  var host = process.env.HTTP_HOST || 'localhost';
+  var port = process.env.HTTP_PORT || '7000';
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/2ec4d5b08584b32f28420a5429edda0eb067a77e